build/Dockerfile

```diff
@@ -17,6 +17,6 @@ RUN CGO_ENABLED=0 make build
 FROM alpine
 
 COPY --from=builder /go/src/bifrost/bifrost /bifrost
-COPY --from=builder /go/src/bifrost/config/config.json config/config.json
+COPY --from=builder /go/src/bifrost/build/config/config.json config/config.json
```

Comment:
Just going to note that we may want two `Dockerfiles` if we ever want to support a more dev-focused docker test build. I totally think this is fine for now though.